### PR TITLE
Deprecate Registration Tokens

### PR DESCRIPTION
**Pull request by brettwp**
This PR relies on the work @bguyza did for #748.  ~~Until that is merged this has a broken link to the Links API.~~  The changes in this PR are to add `Deprecated` to the docs for `registration.tokens` and remove the `tokens` object from the `.json` files.

### CODE REVIEW

File: site/docs/v1/tech/apis/identity-providers/_link-response-body-base.adoc [hunk 1]
```diff
@@ -27,7 +27,7 @@ The token returned from the identity provider. This is treated as an opaque toke
 +
 [NOTE]
 ====
-Prior to version 1.28.0, this value can be retrieved using the link:/docs/v1/tech/apis/registrations.adoc[User Registration APIs]
+Prior to version 1.28.0, this value can be retrieved using the link:/docs/v1/tech/apis/registrations[User Registration APIs]
```
Comment (robotdan) [inline site/docs/v1/tech/apis/identity-providers/_link-response-body-base.adoc:30]: You have to add a trailing slash to this, so it should look like this:

> link:/docs/v1/tech/apis/registrations/[User Registration APIs]

Comment (brettwp) [reply site/docs/v1/tech/apis/identity-providers/_link-response-body-base.adoc]: That is now fixed.

**Comment by robotdan** [comment]
Is this one ready for merge?

**Comment by brettwp** [comment]
Yes, it is.

**Comment by robotdan** [comment]
You should have authority to merge, let me know if you need anything from me.

**Pull request merged**